spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusRuntimeException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.servicebus;
+
+import org.springframework.core.NestedRuntimeException;
+
+/**
+ * The Azure Event Hub specific {@link NestedRuntimeException}.
+ *
+ * @author Warren Zhu
+ */
+public class ServiceBusRuntimeException extends NestedRuntimeException {
+
```

Comment:
Why inheriting from `NestedRuntimeException`? So for every Runtime Exception for Service Bus we will use this exception instead of use different exceptions to distinguish exception causes.

Reply:
`NestedRuntimeException` is handy class provided by Spring for wrapping runtime exceptions with a root cause. We'll throw such exception when we got checked exception thrown by service bus client. We don't want user to explicitly handle service bus related checked exception.